test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.RunTests;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public sealed class NormalizerTests : TestDataPipeBase
+    {
+        public NormalizerTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void NormalizerWorkout()
+        {
+            string dataPath = GetDataPath("iris.txt");
+
+            var loader = new TextLoader(Env, new TextLoader.Arguments
+            {
+                Column = new[]{
+                    new TextLoader.Column("float1", DataKind.R4, 1),
+                    new TextLoader.Column("float4", DataKind.R4, new[]{new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column("float4", DataKind.R4, new[]{new TextLoader.Range(1, 4) }),
```

Comment:
Is it on purpose to have two float4?

Reply:
duh...

---
In reply to: [214495561](https://github.com/dotnet/machinelearning/pull/797#discussion_r214495561) [](ancestors = 214495561)